test/base/MockAuthConnector.scala

```diff
@@ -36,7 +39,7 @@ trait MockAuthConnector extends MockitoSugar with Stubs {
   val eoriWhitelistMock: EoriWhitelist = mock[EoriWhitelist]
 
   val mockAuthAction =
-    new AuthActionImpl(authConnectorMock, eoriWhitelistMock, stubMessagesControllerComponents())
+    new AuthActionImpl(authConnectorMock, eoriWhitelistMock, PlayBodyParsers()(NoMaterializer))(global)
```

Comment:
there is method for bodyParsers in `Helpers`

Reply:
That doesn't work, stub is not enough because somehow we're using this. To use methods from Helpers we will need to add logic to stub that we take from this class. This approach is easier and less problematic.

if there is some reason where this approach is not good I can revert those changes.